pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1188,3 +1188,90 @@ func TestFairEvaluationForNodes(t *testing.T) {
 		}
 	}
 }
+
+func TestFitErrorNoSpammy(t *testing.T) {
+	tests := []struct {
+		name           string
+		nodeNum1       int
```

Comment:
perhaps `errorNodes` and `falseNodes`?

Reply:
Both `FakeFilter` and `FalseFilter` used here will return `unschedulable`, `errorNodes` might not fit here.